src/libs/utils.js

```diff
@@ -590,7 +591,20 @@ export const getSearchFilterFunctions = () => {
         const match = find(isMatch)(matchable);
         return !isNil(match);
       })(targetList);
-    }
+    },
+    datasets: (term, targetList) => filter(dataset => {
+      const duosId = dataset.alias;
+      const dataSubmitter = findPropertyValue(dataset, 'Data Depositor');
+      const datasetName = findPropertyValue(dataset, 'Dataset Name');
+      const dataCustodian = findPropertyValue(dataset, 'Data Depositor');
+      getDataUseCodes(dataset);
```

Comment:
See my comment about lazily enriching.  This will not be performant if this collection includes many datasets that need to call ontology.  The underlying code in getDataUseCodes may need to make multiple requests of ontology to produce the correct set of codes.